hardware-and-software-requirements.md

```diff
@@ -171,7 +172,7 @@ As an open-source distributed SQL database, TiDB requires the following network
 
 | Component | Disk space requirement | Healthy disk usage |
 | :-- | :-- | :-- |
-| TiDB | At least 30 GB for the log disk | Lower than 90% |
+| TiDB | <ul><li>At least 30 GB for the log disk</li> <li> Starting from v6.5.0, `Fast Online DDL` (controlled by variable [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)) is enabled by default to accelerate DDL operations, such as adding indexes. If DDL operations involving large objects exist in the system, it is highly recommended to prepare additional SSD disk space for TiDB (100 GB or higher). For detailed configuration instructions, please refer to [Set temporary space for TiDB instances](/check-before-deployment.md#set-temporary-space-for-tidb-instances-recommended) </li></ul> | Lower than 90% |
```

Comment:
```suggestion
| TiDB | <ul><li>At least 30 GB for the log disk</li> <li> Starting from v6.5.0, `Fast Online DDL` (controlled by variable [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)) is enabled by default to accelerate DDL operations, such as adding indexes. If DDL operations involving large objects exist in the system, it is highly recommended to prepare additional SSD disk space for TiDB (at least 100 GB). For detailed configuration instructions, see [Set a temporary space for a TiDB instance](/check-before-deployment.md#set-a-temporary-space-for-a-tidb-instance-recommended) </li></ul> | Lower than 90% |
```

Reply:
> (at least 100 GB)

100 GB and higher is recommended for most cases. However, space less than 100GB still do its job with less performance.

Reply:
"(at least 100 GB)" implies less space doesn't work.

Reply:
changed to 100 GB or more